testing/tests/DevExpress.ui.widgets.scheduler/integration.appointments.tests.js

```diff
@@ -45,6 +45,10 @@ function getOffset() {
     }
 }
 
+function isDeviceDesktop() {
```

Comment:
You can use it at [L41](https://github.com/DevExpress/DevExtreme/blob/8ebcf75985dae35216690a27c2e6d88abc6eb524/testing/tests/DevExpress.ui.widgets.scheduler/integration.appointments.tests.js#L41)